manifests/moco-nodes.pp

```diff
@@ -1107,6 +1107,22 @@
     include toplevel::server::shipitscriptworker
 }
 
+node /^tb-shipit-dev-\d+\.srv\.releng\..*\.mozilla\.com$/ {
```

Comment:
Nit, not blocking: I think it'd be good to be consisent with these namings. Reading https://hg.mozilla.org/build/puppet/file/default/manifests/moco-nodes.pp I see lots of inconsistencies - but not necessarily for TB, but in general. In this case, I see:

tb-signing
tb-depsigning-worker
tb-balrog-
^tb-balrogworker-dev\
^tb-beetmover-
^tb-beetmover-dev

So we'd rather have them be:
`tb-shipit` (prod)
`tb-shipitworker-dev` 

But I see your set follows beetmover so I guess afterall there's some consistency. 
My own preferences would be to keep the "worker" string within these FQDN to make it clear upfront they are all scriptworkers under the hood.

Reply:
One thing to keep in mind (I ran into this with the treescriptworker), is that there is a limit of 22 characters for worker ids, which we take from the hostname. I do agree that we should be more consistent in general with these names.

I'd suggest, though, that if we want `worker` in the name, we should add another level of name and put it there.

Reply:
Ah, damn, I forgot of that, mea culpa. You're right, +1 to your suggestion.